### PR TITLE
Refactor Project

### PR DESCRIPTION
**Pull request by addisalemseifu**
-make the book list background alternating background color.
-put all the functionalities in a function.
-edit the styling so that it will look like the example.
-did the project solo because I was instructed to do so.
Untitled

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @addisalemseifu 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has no visible contribution from your pair programming partner. Kindly check the Work and Submission mode on the project requirements.
Screenshot 2023-03-22 at 9 11 21 PM




Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — COMMENTED** [review]
Hi Addisalem, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not contributed by all assigned pair group members. Kindly ensure that all assigned members to this project contribute evenly to this task, and then request another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+const addBooks = document.querySelector('.addBook');
+const booksList = document.querySelector('.books-list');
+const remover = document.getElementsByClassName('remover');
+const newB = document.getElementsByClassName('thebook');
+function Book(name, title) {
+  this.name = name;
+  this.title = title;
+  this.id = Math.random();
+}
```
Comment (jasonfelice) [inline script.js]: - [ ] It seems like you are not using JavaScript classes. Kindly, refactor your code to make sure you are using `class`.
_Consider the following example:_
```
class Books {
  constructor() {
    this.books = [];
  }
    //  Here you will define all the methods. Here's an example.
    
    addbook(title, author) {
      // Code to add the book to `this.books`
    }
 }
```
_Remember, this is just an example_

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @addisalemseifu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
No Linter errors✔️
Good Job with the design✔️
Descriptive PR✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (Abenezer-Tilahun) [reply script.js]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @addisalemseifu 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**